### PR TITLE
Add Alamut Institute in Institute settings and use it in Alamut requests

### LINKED ISSUE #3774: Alamut 1.7 appears to require "institution" parameter
Search requests for Alamut seem to stop going through somewhere after Alamut version 1.5.1, but manually adding the ostensibly optional parameter "institution=THATSUS1234" seems to open it ok.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3774 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Go to institute settings and save an alamut link and key for the demo institute
1. Change local scout settings parameter HIDE_ALAMUT_LINK to False
2. Start a local instance of scout and go to a variant
3. Click on the alamut link button and check that the link has the expected format

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN 
- [x] tests executed by DN

### CODE REVIEW

**Comment by northwestwitch** [comment]
image


image

**Comment by dnil** [comment]
✅ Tested locally with Alamut Visual Plus v1.5.1 (closest I had before v1.6) and v1.7 (latest).

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Display dismissed variants comments on general case report
 - Modify ACMG pathogenicity impact (most commonly PVS1, PS3) based on strength of evidence with lab director's professional judgement
 - REViewer button on STR variant page
+- Alamut institution parameter in institute settings for Alamut Plus software
```
Comment (dnil) [inline CHANGELOG.md:15]:
```suggestion
- Alamut institution parameter in institute settings for Alamut Visual Plus software
```

File: tests/server/test_links.py [hunk 2]
```diff
@@ -35,8 +37,9 @@ def test_alamut_link(app, institute_obj, variant_obj):
         link_to_alamut = alamut_link(institute_obj, variant_obj, 38)
         # THEN the link should contain genome build info
         assert "GRCh38" in link_to_alamut
-        # As well as Alamut Visual Plus API key
+        # As well as Alamut Visual Plus API key and institution
```
Comment (dnil) [inline tests/server/test_links.py:40]: 👍

**Review by dnil — APPROVED** [review]

**Pull request merged**